src/main/resources/whitelist

```diff
@@ -90,6 +90,8 @@ method io.gravitee.gateway.api.http.HttpHeaders toSingleValueMap
 
 method io.gravitee.common.util.MultiValueMap containsAllKeys java.util.Collection
 
+method io.gravitee.secrets.api.el.DelegatingEvaluatedSecretsMethods get java.lang.String
+method io.gravitee.secrets.api.el.DelegatingEvaluatedSecretsMethods get java.lang.String java.lang.String
```

Comment:
Maybe a small test case but approved anyway 👍

Reply:
I'm going to add it in the service-secrets where I test all EL expressions, unless you had something else in mind